examples/src/big_compute_buffers/tests.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Empty file? This example definitely needs tests

Reply:
On the fragmentation and pagination notes: 
1) for the fragmentation is a warning similar to what's mentioned in the doc-string on `wgpu::Limits` enough? We could link to some external docs, perhaps https://developer.nvidia.com/docs/drive/drive-os/archives/6.0.4/linux/sdk/common/topics/graphics_content/avoiding_memory_fragmentation.html

2) Perhaps a pagination example is also required that this example could link to? [I would be interested in working on that]

Reply:
> I could use a bit of advice on the test(s) for this: the included shader relies on traversing the contiguous array (made from the multiple buffers) via an OFFSET currently it needs to know this information ahead of compile time, hence the consts.

the first issue seems like it has been fixed?

> Perhaps a pagination example is also required that this example could link to? [I would be interested in working on that]

Isn't this supposed to be an example of pagination?

> for the fragmentation is a warning similar to what's mentioned in the doc-string on wgpu::Limits enough? We could link to some external docs, perhaps

I would just discuss how larger allocations are more prone to memory fragmentation as you always need to find contiguous space for each allocation

Reply:
> > the first issue seems like it has been fixed?
>
> **Fixed**, yes, but it's neither nice, nor idiomatic I was hoping some maintainers may have a 'prettier' solution.

> > Isn't this supposed to be an example of pagination?
>
>  I have been doing CUDA for nearly 7 years, and have never heard it called this, we always call it 'chunking'. Asking around I didn't find anyone who recognised it outside of the context it's usually used in the web spheres for.. but ok.

> >I would just discuss how larger allocations are more prone to memory fragmentation as you always need to find contiguous space for each allocation
>
> I've added some cautionary notes & suggested further reading.